packages/analytics/analytics-chart/src/components/AnalyticsChart.vue

```diff
@@ -429,6 +438,22 @@ const chartTooltipSortFn = computed(() => {
   }
 })
 
+const chartHeaderPosition = computed(() => {
+  return props.chartTitle || !hasKebabMenuAccess || (resultSetTruncated.value && maxEntitiesShown.value) ? 'relative' : 'absolute'
```

Comment:
`resultSetTruncated && maxEntitiesShown` -- I know this is the old logic, but shouldn't this be `||` rather than `&&`?  We need to show a header if either of these warnings is visible?  (Honest question, I may not be up to date on the designs.)

Reply:
I don't remember how this part of explore works, but may be this was based off an old faulty assumption. 😅 

Those computed props are based off the `limit` and `truncated` in the explore response. Is the presence of both required to determine that the results are truncated? If not, then may be we can just use `trancated` ?

Reply:
Aaagh, I'm sorry.  I dug a bit deeper and these two things are complementary: I lost the thread and thought they were two different things.  Technically, you could just use `truncated`, but the current logic is fine.  No need for changes.